### ISSUE TITLE
Plugin UI causes CTD

### ISSUE THREAD

**Issue by VariableVixen** [AtmoOmen/CurrencyTracker#4]
My game is normally stable (at least in that I _rarely_ crash), but within about ten minutes of opening the CurrencyTracker UI (and keeping it open), my game invariably crashes. This has happened multiple times now, with no other determinable common factor.

**Comment by VariableVixen** [comment]
`CurrencyTracker.Windows.Main.ExportToCSV()` [calls`ImGui.EndCombo()` instead of `ImGui.EndPopup()`](https://github.com/AtmoOmen/CurrencyTracker/blob/2b4542aece5957b148f126167b89d893711c9570/CurrencyTracker/Windows/Main.cs#L546C30-L546C30), which may be related.

**Comment by VariableVixen** [comment]
It looks like `CurrencyTracker.Windows.Main.LanguageSwitch()` [does the same thing](https://github.com/AtmoOmen/CurrencyTracker/blob/2b4542aece5957b148f126167b89d893711c9570/CurrencyTracker/Windows/Main.cs#L628).

**Comment by AtmoOmen** [comment]
Thank you for your report!
Since the plugin is still working well on my game client and hasn't caused any crashes, I'll need some time to determine the cause of the crash. Imgui.EndCombo() you mentioned is already shown in my initial version (it's my first time approaching Imgui, and I learned this usage from some code examples, which seemed to work fine). I'll start by replacing all instances of Imgui.EndCombo() with Imgui.EndPopup() and then proceed to carefully review other parts of the code to ensure they are functioning properly!

**Comment by VariableVixen** [comment]
I'm pretty sure that `EndCombo()` is specifically to close `BeginCombo()`, and the `BeginPopup()` calls need `EndPopup()` instead, so only those two should need to be changed. But if you just open the plugin window on your client and leave it there, do you not CTD after a time?

**Comment by AtmoOmen** [comment]
Yes, tbh, just yesterday, I went out for a while while conducting some additional tests on the plugin. I left the plugin interface open during that time, and it remained open for around four hours. When I returned home, the game was still running fine.
But I do believe, that it's certain sections of my plugin's code that be responsible for the crash you experienced. I sincerely apologize for that, and I am diligently working on fixing the code as best as I can!
